contracts/migrations/7_create_token_whitelist.js

```diff
@@ -36,6 +25,17 @@ async function createTokenWhitelistForTestnets() {
   await token.addAllowedTransactor(marketplace.address, { from: tokenOwner })
   console.log(`Added marketplace ${marketplace.address} to whitelist`)
 
+  if (network === 'mainnet' || process.env['SIMULATE_MAINNET']) {
+    const addresses = [
```

Comment:
nit: maybe add a comment that this is for adding the affiliate and ERC20 multisig wallet.

Reply:
done